src/dhali/payment_claim_generator.py

```diff
@@ -0,0 +1,132 @@
+# create a network client
+from xrpl.clients import JsonRpcClient
+from xrpl.models.transactions import PaymentChannelCreate
+import xrpl
+from xrpl.wallet import generate_faucet_wallet, Wallet
+import json
+
+
+client = JsonRpcClient("https://s.altnet.rippletest.net:51234/")
+total_xrp_in_channel = "100000000"
+cancel_after = 1000
+settle_delay = 15768000
```

Comment:
These seem `TODO`-worthy, just so we don't miss them later.  The last three variables seem like they might be configuration